src/code/codeManager.js

```diff
@@ -25,39 +25,43 @@ CodeManager.prototype.resolveStep = function (stepIndex, tx) {
   this.trigger('resolvingStep')
   var self = this
   if (stepIndex === 0) {
-    self.getCode(tx.to, stepIndex, tx)
+    self.retrieveCodeAndTrigger(tx.to, stepIndex, tx)
   } else {
     this.traceManager.getCurrentCalledAddressAt(stepIndex, function (error, address) {
       if (error) {
         console.log(error)
       } else {
-        self.getCode(address, stepIndex, tx)
+        self.retrieveCodeAndTrigger(address, stepIndex, tx)
       }
     })
   }
 }
 
-CodeManager.prototype.getCode = function (address, currentStep, tx) {
+CodeManager.prototype.retrieveCodeAndTrigger = function (address, stepIndex, tx) {
   var self = this
+  this.getCode(address, function (error, result) {
+    if (!error) {
+      self.retrieveIndexAndTrigger(address, stepIndex, result.code)
+    }
```

Comment:
Please at least log the error, but in general, I would avoid using console.log because we do not know whether we are in a browser context or not. Perhaps this has to be refactored in the future.